Cargo.toml

```diff
@@ -27,7 +27,7 @@ all-features = true
 features = ["serde", "u128", "v1", "v3", "v4", "v5"]
 
 [dependencies]
-serde = { version = "1.0.16", optional = true, default-features = false }
+serde = { version = "1", optional = true, default-features = false }
```

Comment:
We can't use the general "1" as a dependency specifier here, since we are depending on a particular "1.0.56" version here for 128 (even though it is optional)

Better to use "1.0.56"

Reply:
done